### ISSUE TITLE
Skipping Rows during navigation

### ISSUE THREAD

**Issue by MarcM0** [open-navigation/opennav_coverage#66]
Given a small path like this
![Screenshot from 2024-07-11 14-15-28](https://github.com/open-navigation/opennav_coverage/assets/30278842/3f321f97-c05e-4b41-a899-cff25eeba825)
The robot will take a trail that looks like this
![image](https://github.com/open-navigation/opennav_coverage/assets/30278842/ac9637de-c4e8-4a49-b938-b4f2215e8f7a)
It seems it wants to take shortcuts

**Comment by MarcM0** [comment]
Upon further inspection, it seems this mostly happens if the turning radius is too small

**Comment by SteveMacenski** [comment]
Yeah, probably just a tuning thing. I asked about algorithms because if you're using something like a RPP where its following a carrot, if you have the  carrot distance too long, then it might be bypassing some of the field. That's not much an issue for big fields, but if the scale of this is really small, then it would be something you need to more carefully tune since short cutting any amount of the row would be most of the row.
